dask/dataframe/tseries/resample.py

```diff
@@ -30,12 +31,20 @@ def _resample_series(
     out = getattr(series.resample(rule, **resample_kwargs), how)(
         *how_args, **how_kwargs
     )
-    return out.reindex(
+    new_index = (
```

Comment:
Just for readability, can you put this in a regular if / else block rather than a ternary expression?

Reply:
Fixed